REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.6
-Compat 0.18
+Compat 0.27 # for Sys.iswindows
```

Comment:
It has happened to me a couple of times now that when I release a new version of a package, Tony Kelman goes through and checks that the Compat version isn't higher than it needs to be (I guess to avoid package resolution issues, since so many packages depend on Compat). This is just to make his job a little easier.